### ISSUE TITLE
[Accessibility] The curly braces " { }" does not meet required minimum color contrast ratio with its background.

### ISSUE THREAD

**Issue by Krituhar** [microsoft/vscode#227499]
- VS Code Version: 1.88.0
- OS Version: MacOS Sonoma (v14.6.1)

**[Issue]**
The "{" and "}" text that does not meet the required minimum color contrast ratio with its background.

Foreground color: #319331
Background color: #FFFFFF
Contrast ratio: 3.93:1
Required contrast ratio for this content: 4.50:1

**[User Impact]**
Users with low vision will have difficulty reading this content.

**[Code Reference]**
```
<span class="mtk1 bracket-highlighting-1">{</span>
```

**[Recommendation]**
Ensure text and images of text provide sufficient contrast. The following contrast ratios are required:

- Text smaller than 18 pt (24 px), or smaller than 14 pt (19 px) if bold, must have a color contrast ratio of 4.50:1 or more with adjacent colors.

- Text 18 pt (24 px) or larger, or 14 pt (19 px) or larger if bold, must have a color contrast ratio of 3.00:1 or more with adjacent colors.

Disabled controls that do not accept user interaction are exempt from this requirement.

To calculate color contrast ratios, use a tool such as the Level Access Accessible Color Picker Chrome extension: https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld or the Color Contrast Checker: https://www.levelaccess.com/color-contrast-checker-new/

Screenshot of issue:
![Screenshot 2024-09-03 at 5 42 58 PM](https://github.com/user-attachments/assets/23358fd0-035e-4de6-88d5-a224a60348f4)

**Comment by meganrogge** [comment]
Closing because we have high contrast themes, and no users have reported this as an issue.
